### PR TITLE
chore: aliasing substrapp django models

### PR DESCRIPTION
**Pull request by sergebouchut2**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202951505377093

### CODE REVIEW

File: backend/api/tests/views/test_views_model.py [hunk 1]
```diff
@@ -50,12 +50,12 @@ def setUp(self):
         algo = factory.create_algo()
         compute_plan = factory.create_computeplan()
 
-        self.train_task = factory.create_computetask(compute_plan, algo, category=computetask_pb2.TASK_TRAIN)
-        simple_model_1 = factory.create_model(self.train_task, category=ModelRep.Category.MODEL_SIMPLE)
-        simple_model_2 = factory.create_model(self.train_task, category=ModelRep.Category.MODEL_SIMPLE)
+        self.train_task = factory.create_computetask(compute_plan, algo, category=ComputeTask.Category.TASK_TRAIN)
```
Comment (sergebouchut2) [inline backend/api/tests/views/test_views_model.py:53]: We should use values stored in the model.

**Comment by Owlfred** [comment]
End to end tests: :hourglass_flowing_sand: **STARTING**

**Review by Kelvin-M — APPROVED** [review]
It seems good ! 
I didn't review completely as it is repetitive replacements.
E2E tests should catch any errors as it's modifications on imports.

**Pull request merged**